hadoop-cloud/README.md

```diff
@@ -16,5 +16,5 @@ Integration tests will have some extra configurations for example selecting the
 run the test against. Those configs are passed as environment variables and the existence of these
 variables must be checked by the test.
 Like for `AwsS3AbortableStreamBasedCheckpointFileManagerSuite` the S3 bucket used for testing
-is passed in the `S3A_PATH` and the credetinals to access AWS S3 are AWS_ACCESS_KEY_ID and
+is passed in the `S3A_PATH` and the credentials to access AWS S3 are AWS_ACCESS_KEY_ID and
```

Comment:
This is fine. Mind taking a look and see other typos since we're here? Let's fix them in batch in this PR.

Reply:
Yes, of course, I will contribute more.  Actually this typo takes me several hours to locate it.  In the future, I would like to make more technical contributions.  Let me participate in,  let's work together to make spark more beautiful.